src/api/v1beta1/properties.go

```diff
@@ -387,3 +399,10 @@ func splitArg(arg string) (key, value string) {
 	value = split[1]
 	return
 }
+
+func getImageTag(imageURI string) string {
+	if !strings.Contains(imageURI, ":") {
+		return ""
+	}
+	return strings.Split(imageURI, ":")[1]
```

Comment:
If an image with a digest is supplied, that would also break the label.
Then the tag returned would be something like `sha256@<hash>` and an `@` is not allowed in a label value either.
I'd suggest normalizing the string.
E.g., any character not matching a regex is replaced with a dash or removed or something similar.